.gitlab-ci/terraform.yml

```diff
@@ -115,6 +114,7 @@ tf-apply-ovh:
     PROVIDER: openstack
     CLUSTER: $CI_COMMIT_REF_NAME
     ANSIBLE_TIMEOUT: "60"
+    SSH_USER: ubuntu
```

Comment:
nit: ubuntu ssh user should be in ovh_variables

Reply:
Not yet, since the CoreOS image will surely use another `SSH_USER`.